src/batch/normalization/services/computeLabelStatus.spec.ts

```diff
@@ -72,7 +72,7 @@ describe('updateLabelStatus', () => {
     it(`when decision codeNAC is in transmissible CodeNAC list`, () => {
```

Comment:
is in transmissible codeDecision list ?

Reply:
Oui j'ai pris le fichier en diagonal je vais me pencher dessus voir si j'ai pas raté autre chose